2006Scape Client/src/main/java/Main.java

```diff
@@ -35,10 +35,36 @@ public static void main(String[] args) {
 						Game.customSettingVisiblePlayerNames = true;
 						Game.customSettingMinItemValue = 1000;
 						break;
+					case "-fixesnooverride":
+						ClientSettings.CUSTOM_SETTINGS_TAB = true;
+						ClientSettings.BILINEAR_MINIMAP_FILTERING = true;
+						ClientSettings.FIX_TRANSPARENCY_OVERFLOW = true;
+						ClientSettings.FULL_512PX_VIEWPORT = true;
+						break;
 					case "-no-nav":
-					case"-disable-nav":
+					case "-disable-nav":
 						ClientSettings.SHOW_NAVBAR = false;
 						break;
+					case"-no-snow":
+					case"-hide-snow":
+					case"-disable-snow":
+						ClientSettings.SNOW_FLOOR_ENABLED = false;
+						ClientSettings.SNOW_FLOOR_FORCE_ENABLED = false;
+						ClientSettings.SNOW_OVERLAY_FORCE_ENABLED = false;
+						ClientSettings.SNOW_OVERLAY_ENABLED = false;
+						break;
+					case"-no-roofs":
+					case"-hide-roofs":
+					case"-disable-roofs":
+						ClientSettings.HIDE_ROOFS = true;
+						break;
+					case"-show-zoom":
+						ClientSettings.SHOW_ZOOM_LEVEL_MESSAGES = true;
+						break;
+					case"-crtl-key-zoom":
+					case"-control-key-zoom":
+						ClientSettings.CONTROL_KEY_ZOOMING = true;
```

Comment:
should we enable this with -qol aswell?

Reply:
We could, yeah, but I was thinking since it's kind of a separate feature it could be it's own flag, that way it's not forced.

Reply:
i think we should enable it by default with -qol but just have a command in-game to enable/disable on the fly